recipes/mpfr/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export LD_LIBRARY_PATH=$PREFIX/lib:$LD_LIBRARY_PATH
```

Comment:
I suppose we could uninstall these things from our CI, but these are real problems that could occur when building locally on a user's system. So, it isn't bad that we come up with a fix.